src/coreclr/gc/unix/numasupport.cpp

```diff
@@ -9,9 +9,12 @@
 #include <stdio.h>
 #include <dirent.h>
 #include <string.h>
-#include <sys/syscall.h>
 #include <minipal/utils.h>
 
+#ifdef TARGET_LINUX
+#include <sys/syscall.h>
+#endif
```

Comment:
Is this used by other Unix systems?

Reply:
No, it is only used on Linux:

https://github.com/dotnet/runtime/blob/e5c631d28b985b773b28d0cb2346f49a92bd41dd/src/coreclr/gc/unix/numasupport.cpp#L52-L63

and 

https://github.com/dotnet/runtime/blob/e5c631d28b985b773b28d0cb2346f49a92bd41dd/src/coreclr/gc/unix/numasupport.cpp#L81-L83